style-sheets/primevue-components-overrides.scss

```diff
@@ -391,7 +392,57 @@ label {
 .#{variables.$primevue-prefix}-card .#{variables.$primevue-prefix}-card-footer {
     padding: 0;
 }
+
 /* ------------------------------------ primevue table --------------------------------- */
+
+.custom-access-table {
+    background: transparent;
+    border-radius: 0.25rem 0.25rem 0 0;
+    border: 0.125rem solid map.get(lightTheme.$light-theme, 'border-subtle-00');
+}
+
+.custom-table-header {
```

Comment:
This table header is what we added for FAM, primevue data table component doesn't have a header, so we shouldn't put it here. Let's put it back to our FAM repo.